Swift Vocabulary/Swift Vocabulary/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,100 @@
+                                            <nil key="textColor"/>
+                                            <nil key="highlightedColor"/>
+                                        </label>
+                                    </subviews>
+                                </tableViewCellContentView>
+                                <connections>
+                                    <segue destination="BYZ-38-t0r" kind="show" identifier="ShowDefinitionSegue" id="xpQ-Jb-75c"/>
+                                </connections>
+                            </tableViewCell>
+                        </prototypes>
+                        <connections>
+                            <outlet property="dataSource" destination="Iei-Pz-j7w" id="SKo-nE-CH0"/>
+                            <outlet property="delegate" destination="Iei-Pz-j7w" id="Vxx-F2-3yN"/>
+                        </connections>
+                    </tableView>
+                    <navigationItem key="navigationItem" title="Swift Vocabulary" id="nyS-1t-2XO"/>
+                </tableViewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="qjn-3j-DeM" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="-526.08695652173913" y="184.82142857142856"/>
+        </scene>
+        <!--Definition View Controller-->
+        <scene sceneID="tne-QT-ifu">
+            <objects>
+                <viewController id="BYZ-38-t0r" customClass="DefinitionViewController" customModule="Swift_Vocabulary" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="8bC-Xf-vdC">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" fixedFrame="YES" text="some words" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="SQe-GW-DeC">
+                                <rect key="frame" x="88" y="173" width="209" height="255"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                <accessibility key="accessibilityConfiguration" identifier="lblDefinition"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                <nil key="textColor"/>
+                                <nil key="highlightedColor"/>
+                            </label>
```

Comment:
Again, make sure you follow the directions. This label was supposed to be constrained. Also there was supposed to be a textView. Technically your project doesn't meet the requirements of the assignment currently, so I would like for you to fix this and let me know when you have it done. 

> On the definition view controller, add a label for the word, and below that, a text view for the definition.
Use Add Missing Constraints to add constraints.